docs/releases.md

```diff
@@ -0,0 +1,53 @@
+# Releases
+
+This document describes how the release flow looks like and how to create a marketplace release.
+
+## Release process
+
+The release process is based on Git tags. Once tag is created the release process will start automatically.
```

Comment:
Once tag is created the release process will start automatically. -> Once a tag is created, the release process starts automatically.